.github/workflows/build.yaml

```diff
@@ -0,0 +1,37 @@
+name: Build
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Set up JDK 17
+        uses: actions/setup-java@v3
+        with:
+          java-version: "17"
+          distribution: "temurin"
+          cache: gradle
+
+      - name: Make Gradle wrapper executable
+        run: chmod +x ./gradlew
```

Comment:
Can just update the file mode in git instead

Reply:
I'm on Windows though, and my Linux VM broke a few days ago 😄 Maybe you can change the file permissions on your system and PR the updated file to my branch? Or maybe @sfPlayer1 can do it himself, I'm not sure if he's on Linux

Reply:
You can run
`git update-index --chmod +x gradlew`.
chmod doesn't change the file in the git repo.

Reply:
Nice, thanks!